packages/auth/graphql/resolvers/User.js

```diff
@@ -20,5 +21,13 @@ module.exports = {
       return user.roles
     }
     return []
+  },
+  createdAt (user, args, { user: me }) {
+    Roles.ensureUserIsMeOrInRoles(user, me, userAccessRoles)
```

Comment:
For me it's unclear when we should throw based on field level access control. Who will consume `createdAt` and `updatedAt`? I can see use cases in the public profile for `createdAt`. Maybe just make it nullable and return null when not authorised?

Reply:
disagree. IMHO the schema should represent reality. Every user has a createdAt and an updatedAt date. I think we should continue to treat access violations with throwing an exception. The client is responsible to only request the appropriate fields, in this case it's the admin tool.